.github/workflows/source/hasEOLwhiteSpace

```diff
@@ -12,6 +12,8 @@
 # note: .md files can contain EOL white spaces for syntax reasons
 #       (newline without starting a new paragraph)
 
+set -eu -o pipefail
```

Comment:
Just for general knowledge: this is a very useful `bash` command that allows the script to indicate in which line it broke, if there are any issues in it.

Reply:
oh this whole PR is still very much WIP and will take a while :D